org.eclipse.nebula.widgets.nattable.extension.poi.feature/feature.xml

```diff
@@ -19,48 +19,21 @@
       %license
    </license>
 
-   <plugin
-         id="org.eclipse.nebula.widgets.nattable.extension.poi"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.apache.poi"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.apache.commons.math3"
-         version="0.0.0"/>
-
-   <plugin
-         id="org.apache.commons.commons-collections4"
-         version="0.0.0"/>
+   <requires>
+      <import plugin="org.apache.commons.commons-collections4"/>
+      <import plugin="org.apache.commons.commons-compress"/>
+      <import plugin="org.apache.commons.commons-io"/>
+      <import plugin="org.apache.logging.log4j.api"/>
+      <import plugin="org.apache.commons.math3"/>
+      <import plugin="org.apache.xmlbeans"/>
+      <import plugin="org.apache.poi"/>
+      <import plugin="org.apache.poi.ooxml"/>
+      <import plugin="org.apache.poi.ooxml.schemas"/>
+      <import plugin="com.zaxxer.sparsebits"/>
+   </requires>
```

Comment:
At least some here seem not needed as they are already transitively required.

Reply:
Move all such 3rd party things to a feature not consumed by end users.

Reply:
Does it need to be a feature, or can those dependencies also be added as bundles in a new category?